test/__snapshots__/test.js.snap

```diff
@@ -45,6 +46,9 @@ function third(){
   "something.css": "#id {
     background: white;
 }",
+  "with-broken-sourcemap.js": "function meaningOfLife(){throw new Error(42)}
+//# sourceMappingURL=with-broken-sourcemap.map",
+  "with-broken-sourcemap.map": "{\\"version\\":3,\\"sources\\":[\\"0\\"],\\"names\\":[\\"meaningOfLife\\",\\"Error\\"],\\"mappings\\":\\"AAAA,SAASA,gBAEP,MAAM,IAAIC,MADa\\",\\"file\\":\\"with-broken-sourcemap.js\\"}",
```

Comment:
Is there a way we can output `"sources: ["with-broken-sourcemap.js]` instead of `sources: ["0"]`?

Reply:
I don't know. can you explain why that would be useful or more correct than what we currently have?